dialogue-clients/src/main/java/com/palantir/dialogue/clients/DialogueClients.java

```diff
@@ -50,6 +50,22 @@ public static ReloadingFactory create(Refreshable<ServicesConfigBlock> scb) {
                 ImmutableReloadingParams.builder().scb(scb).build(), ChannelCache.createEmptyCache());
     }
 
+    /**
+     * Prefer {@link #create(Refreshable)}. Using a configured {@link ReloadingFactory} should always be preferred
+     * over this method, it only exists to ease migration.
+     *
+     * Construct an instance of the given {@code clientInterface} which can be used to make network calls to the
+     * single conceptual upstream identified by {@code clientConfiguration}.
+     *
+     * Behaviour is undefined if {@code clientConfiguration} contains no URIs.
+     *
+     * @param clientInterface Dialogue client interface annotated with {@link com.palantir.dialogue.DialogueService}
+     * @param clientConfiguration Configuration which <b>must</b> contain a user-agent
+     */
+    public static <T> T create(Class<T> clientInterface, ClientConfiguration clientConfiguration) {
```

Comment:
createLegacy? some other way to indicate people shouldn't really be doing this? I guess you have docs

Reply:
Ya, I went back and forth on naming a few times here, but landed on `create` matching `jaxrsclient` becuase it's a direct replacement, and we don't provide alternatives for some use-cases (when trust material exists only in memory, for example).

Reply:
To clarify, I'm not confident I've picked the right name, just recording my thought process :-)